templates/zerver/base.html

```diff
@@ -33,7 +33,9 @@
         {% block content %}
         {% endblock %}
 
-        <div hidden id="page-params" data-params='{{ page_params|default({"debug_mode": false})|tojson }}'></div>
+        {% set all_page_params = default_page_params.copy() %}
+        {% set _ = all_page_params.update(page_params) %}
+        <div hidden id="page-params" data-params='{{ all_page_params|tojson }}'></div>
```

Comment:
I guess `page_params` doesn't have a ton of keys, so the copy type work here is cheap.